client/components/plans/plan-icon/index.jsx

```diff
@@ -41,18 +29,14 @@ export default class PlanIcon extends Component {
 
 	render() {
 		const { plan } = this.props;
-		switch ( plan ) {
-			case PLAN_PERSONAL:
-			case PLAN_JETPACK_PERSONAL:
-			case PLAN_JETPACK_PERSONAL_MONTHLY:
+		switch ( true ) {
+			case isPersonalPlan( plan ):
 				return this.getIcon( 'personal' );
-			case PLAN_PREMIUM:
-			case PLAN_JETPACK_PREMIUM:
-			case PLAN_JETPACK_PREMIUM_MONTHLY:
+
+			case isPremiumPlan( plan ):
 				return this.getIcon( 'premium' );
-			case PLAN_BUSINESS:
-			case PLAN_JETPACK_BUSINESS:
-			case PLAN_JETPACK_BUSINESS_MONTHLY:
+
+			case isBusinessPlan( plan ):
```

Comment:
Minor style note: I think it is probably a bit clearer to write this as `if`/`else if`/`else`.

Reply:
I find the switch more readable:

<img width="401" alt="zrzut ekranu 2018-03-12 o 11 19 57" src="https://user-images.githubusercontent.com/205419/37278249-6cd982a4-25e7-11e8-98f0-6c07f6de5f56.png">

Reply:
How about using only `if` blocks?
```
if ( isPersonalPlan( plan ) ) {
  return this.getIcon( 'personal' );
}
if ( isPremiumPlan( plan ) ) {
  return this.getIcon( 'premium' );
}
if ( isBusinessPlan( plan ) ) {
  return this.getIcon( 'business' );
}

return this.getIcon( 'free' );
```

Reply:
Great idea!